rise-financial-es6.js

```diff
@@ -108,14 +109,24 @@
       }, 60000 );
     }
 
+    _onDataPingReceived() {
+      this._dataPingReceived = true;
+
+      if ( this._goPending ) {
```

Comment:
@stulees I don't think we need `this._goPending` variable. The `go` function is already doing a check to ensure that the Display ID, instruments and data ping are received before trying to fetch data. If not, it just returns. So it looks to be unnecessary.